### ISSUE TITLE
show when search for "About"

### ISSUE THREAD

**Issue by ChildishGiant** [elementary/switchboard-plug-about#219]
## What Happened
![Screenshot from 2021-06-24 15 32 11](https://user-images.githubusercontent.com/13716824/123291148-5ef91d80-d501-11eb-8b13-a70825f91e81.png)
There's no about section

## Expected Behavior
About should be visible under administration

## Steps to Reproduce
1. Fresh install from `elementaryos-6.0-daily.20210624`

## Platform Information
* elementaryos-6.0-daily.20210624
* ThinkPad T460 
*  Kernel: 5.8.0-59-generic

**Comment by vjr** [comment]
I believe it's renamed to "System" in the UI, see the bottom portion...

**Comment by danrabbit** [comment]
Yes, this page is now called "System" and is visible in your screenshot :)

**Comment by ChildishGiant** [comment]
:facepalm: Sorry for that. Maybe it should show up when searching for About in the applications menu?

**Comment by danrabbit** [comment]
Sure thing. Reopening so we can track that :)
